asllib/doc/ASLTypingReference.tex

```diff
@@ -1037,6 +1443,182 @@ \subsection{Prose}
 
 \begin{emptyformal}
     \subsection{Formally}
+
+\todocomment{
+Roman: consider defining an ancestor relation and then having the \lca\ be naturally defined as the least common ancestor.
+}
```

Comment:
yes please make a proposal as part of this PR, as discussed

Reply:
I'm deferring this, since we have to check with the work group what is the intended semantics in this case.